javascript.js

```diff
@@ -96,3 +96,36 @@ contactForm.addEventListener('submit', (sub) => {
     sub.preventDefault();
   }
 });
+
+// Data browser
+
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const textInput = document.getElementById('message');
+
+function storeForm() {
+  const data = {
+    name: nameInput.value,
+    email: emailInput.value,
+    text: textInput.value,
+  };
+  localStorage.setItem('data', JSON.stringify(data));
+}
+
+nameInput.addEventListener('input', () => {
+  storeForm();
+});
+emailInput.addEventListener('input', () => {
+  storeForm();
+});
+textInput.addEventListener('input', () => {
+  storeForm();
+});
+
+window.addEventListener('load', () => {
+  const windowData = localStorage.getItem('data');
+  const parseWindow = JSON.parse(windowData);
+  nameInput.value = parseWindow.name;
+  emailInput.value = parseWindow.email;
+  textInput.value = parseWindow.text;
```

Comment:
[OPTIONAL] I think you should try to handle errors here. Why I mean by that, is that sometimes localStorage won't be available, for instance, in some private modes like "incognito" in Chrome.

Reply:
[OPTIONAL] The contrast in the fields from your form is a bit low, I would consider increasing the size of the letter and making it darker. You can specify the styles of the placeholder making use of its Pseudo-Class. [You can read more about here!](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder)

![image](https://user-images.githubusercontent.com/79114233/141507446-79dc8fe8-7b13-4790-8916-691fae2092b5.png)

Reply:
[OPTIONAL] Nowadays it's preferable to add the `defer` attribute to the scripts tags. This ensures that the script is run exclusively after the HTML document has finished loading and can avoid a bug or two from time to time. [Read more about here.](https://www.w3schools.com/tags/att_script_defer.asp) :slightly_smiling_face: 

`<script src="javascript.js" defer></script>`